apps/ssr-tests-v9/src/utils/buildAssets.ts

```diff
@@ -0,0 +1,60 @@
+import { TsconfigPathsPlugin } from '@esbuild-plugins/tsconfig-paths';
+import { build, BuildOptions } from 'esbuild';
+import * as path from 'path';
+
+const NODE_MAJOR_VERSION = process.versions.node.split('.')[0];
+
+const commonOptions: BuildOptions = {
+  bundle: true,
+  jsx: 'transform',
+  loader: {
+    '.svg': 'dataurl',
+  },
+  // Used to resolve packages aliases in stories files.
+  plugins: [TsconfigPathsPlugin({ tsconfig: path.resolve(__dirname, 'tsconfig.base.json') })],
+};
+
+type BuildConfig = {
+  cjsEntryPoint: string;
+  cjsOutfile: string;
+  esmEntryPoint: string;
+  esmOutfile: string;
+};
+
+export async function buildAssets(config: BuildConfig): Promise<void> {
+  const { cjsEntryPoint, cjsOutfile, esmEntryPoint, esmOutfile } = config;
+
+  try {
+    // Used for SSR rendering, see renderToHTML.js
+    await build({
+      ...commonOptions,
+
+      entryPoints: [cjsEntryPoint],
+      outfile: cjsOutfile,
+
+      // External dependencies should not be bundled
+      external: ['@griffel/core', '@griffel/react', 'react', 'react-dom', 'scheduler'],
+      format: 'cjs',
+      target: `node${NODE_MAJOR_VERSION}`,
+    });
+
+    // Used in generated bundle that will be server by a browser
+    await build({
+      ...commonOptions,
+
+      entryPoints: [esmEntryPoint],
+      outfile: esmOutfile,
+
+      inject: [require.resolve('../shims/module')],
```

Comment:
Basically because of this:

![image](https://user-images.githubusercontent.com/14183168/172577420-9bbb8cf4-c6d0-4ba5-973a-668379ba705e.png)

I added the comment.